fmriprep/workflows/base.py

```diff
@@ -285,7 +285,7 @@ def init_single_subject_wf(subject_id):
     # Overwrite ``out_path_base`` of smriprep's DataSinks
     for node in workflow.list_node_names():
         if node.split('.')[-1].startswith('ds_'):
-            workflow.get_node(node).interface.out_path_base = 'fmriprep'
+            workflow.get_node(node).interface.out_path_base = ""
```

Comment:
I believe the whole loop can go away.

Reply:
I don't think so. These nodes will have another default `out_path_base`.